map/gps_track.hpp

```diff
@@ -101,4 +103,7 @@ class GpsTrack final
   bool m_threadExit; // need exit thread
   bool m_threadWakeup; // need wakeup thread
   std::condition_variable m_cv;
+public:
+  std::vector<location::GpsTrackInfo> GetCurrentTrack() const;
+  bool IsCollectionInit() const;
```

Comment:
If this method is really needed:
```suggestion
  bool IsTrackerInitialized() const;
```

Reply:
@kavikhalique WDYT should we remove this check?